src/contentful/utils.js

```diff
@@ -36,10 +42,22 @@ export const getContentfulRequest = (type, contentQuery) => {
   };
 };
 
+const hasValidationErrors = (payload) => payload?.errors;
+
+export const createContentfulRequest = async(type, contentQuery) => {
+  const payload = await createRequest(getContentfulRequest(type, contentQuery));
+  if (hasValidationErrors(payload)) throw new Error('Contentful request contains validation errors');
+
+  return payload;
+};
+
 export const formatImage = (image, assets) => {
   const ref = assets[image.sys.id];
-  if (!ref) return null;
-  return `https:${ref.fields.file.url}`;
+  // optional chaining saves page crash when contentful content in staging spaces is missing
+  const url = ref?.fields?.file?.url;
```

Comment:
do we need elias fix now anymore?

Reply:
As far as I can remember - yes. We had cases in which there were no errors but the url wasn't available. Better to keep it.